tasks/mutant.rake

```diff
@@ -1,18 +1,13 @@
 desc 'Runs mutant'
 task :mutant do
   command = <<-EOS
-    branch_name=$(git symbolic-ref HEAD 2>/dev/null | cut -d"/" -f 3)
-    if [ "$branch_name" != "master" ]; then
-      RUBY_THREAD_VM_STACK_SIZE=64000\
-      bundle exec mutant\
-        --include lib\
-        --require reek\
-        --use rspec\
-        --since master\
-        --jobs 4 'Reek*'
-    else
-      echo "!!! Not running mutant on master branch !!!"
-    fi
+    RUBY_THREAD_VM_STACK_SIZE=64000\
+    bundle exec mutant\
+      --include lib\
+      --require reek\
+      --use rspec\
+      --since master^\
```

Comment:
I think there will be little difference in practice compared to just using master. For branches based off latest master, there is one extra diff being taken into account (and that should be mutant-clean, given it's on master).

The only time this will be a problem is when we merge a branch mutant doesn't like.